starport_template/pubspec.yaml

```diff
@@ -10,16 +10,18 @@ dependencies:
   flutter:
     sdk: flutter
 
-  mobx: ^2.0.3
+  mobx: 2.0.3
   alan:
   uuid: 3.0.4
-  flutter_mobx: ^2.0.1
+  http: 0.13.3
+  decimal: 1.3.0
+  flutter_mobx: 2.0.1
   cosmos_ui_components:
 #    path: ../packagefs/cosmos_ui_components
     git:
       url: https://github.com/tendermint/flutter.git
       path: packages/cosmos_ui_components
-      ref: main
+      ref: wallet_details_page
```

Comment:
@andrzejchm 
I changed this because I had updated a model inside `cosmos_ui_components` in my same branch in order to match the `walletId` parameter.
As soon as we merge this, I will create another PR putting it back to main branch.